languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10069,6 +10069,35 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rulegroup id="IL_FAUT" name="Il faut" default="temp_off">
+          <rule>
```

Comment:
Now that you have made the second rule for "faut pas" etc., you need to make sure that the first rule doesn't match (AP: faut pas|plus).

Also, what do you say we make this rule more powerful by writing `<token inflected="yes" postag="V.*3.*">falloir</token>` or something like that? That way, we can also catch sentences like "Faudrait savoir s'il viendra ou pas" or "Fallait pas insister..."